example/example.cpp

```diff
@@ -1785,59 +1785,658 @@ void deserializeGridHyperslabbing(common::EpcDocument & pck)
 	cout << endl << "END: IJK GRID REP (hyperslabbing)" << endl;
 }
 
+/**
+ * Display the cell geometry of an IJK grid block.
+ * @param ijkGrid			An IJK grid.
+ * @param iInterfaceStart	Sarting i interface of the block.
+ * @param iInterfaceEnd		Ending i interface of the block.
+ * @param jInterfaceStart	Sarting j interface of the block.
+ * @param jInterfaceEnd		Ending j interface of the block.
+ * @param kInterfaceStart	Sarting k interface of the block.
+ * @param kInterfaceEnd		Ending k interface of the block.
+ * @param xyzPoints			The geometry of the block.
+ */
+void displayBlockCellGeometry(AbstractIjkGridRepresentation* ijkGrid,
+	unsigned int iInterfaceStart, unsigned int iInterfaceEnd,
+	unsigned int jInterfaceStart, unsigned int jInterfaceEnd,
+	unsigned int kInterfaceStart, unsigned int kInterfaceEnd,
+	double* xyzPoints
+	)
+{
+	if (xyzPoints == nullptr)
```

Comment:
Hi Mathieu,
As a convention, in the future, try to always put curly bracket with an if please. Even if this is just an one line "if".